libs/core/pyproject.toml

```diff
@@ -17,6 +17,7 @@ jsonpatch = "^1.33"
 PyYAML = ">=5.3"
 packaging = "^23.2"
 jinja2 = { version = "^3", optional = true }
+chevron = "^0.14.0"
```

Comment:
chevron hasn't been updated in 3 years. I don't think we this should be a required dependency, especially since I doubt most users will use mustache for templates. https://github.com/noahmorrison/chevron?tab=readme-ov-file#chevron-is-python-2-and-3-compatible